db/seeds.rb

```diff
@@ -186,3 +186,11 @@
   r.location = boulder
   r.feed_format = "video"
 end.save!
+
+if Rails.env.development?
+  AdminUser.create!(
+    email: 'admin@example.com',
+    password: 'password',
+    password_confirmation: 'password'
+  )
```

Comment:
So this creates an admin user and their credentials are stored into the database?